src/index.html

```diff
@@ -20,8 +20,343 @@
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card cards__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="iMac A1419"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card cards__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="iMac A1419"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card cards__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="iMac A1419"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">Product code: 195434</div>
```

Comment:
The product code '195434' is repeated here. Each product card should have a unique product code to ensure clarity and proper functionality.